crates/ruff_linter/src/rules/flake8_bugbear/rules/mutable_argument_default.rs

```diff
@@ -165,33 +166,38 @@ fn move_initialization(
     content.push_str(stylist.line_ending().as_str());
 
     // Indent the edit to match the body indentation.
-    let content = textwrap::indent(&content, indentation).to_string();
-
-    let initialization_edit = if is_docstring_stmt(statement) {
-        // If the first statement in the function is a docstring, insert _after_ it.
-        if let Some(statement) = body.next() {
-            // If there's a second statement, insert _before_ it, but ensure this isn't a
-            // multi-statement line.
-            if indexer.in_multi_statement_line(statement, locator) {
-                return None;
+    let mut content = textwrap::indent(&content, indentation).to_string();
+
+    // Find the position to insert the initialization after docstring and imports
+    let mut pos = locator.line_start(statement.start());
+    while let Some(statement) = body.next() {
+        // If the statement is a docstring or an import, insert _after_ it.
+        if is_docstring_stmt(statement)
+            || statement.is_import_stmt()
+            || statement.is_import_from_stmt()
```

Comment:
@Hoxbro - I collapsed these two cases -- I think semantically they're the same? We want to skip docstrings and imports, and find the first opportunity to insert after them. (If there's no such opportunity, we insert at the start of the function.)

Reply:
Of course, please let me know if I've misunderstood or messed something up -- happy to fix in a follow-up.

Reply:
No, that sounds right to me.